packages/ui-toolkit/stories/ToggleSwitch.stories.tsx

```diff
@@ -53,4 +57,4 @@ Custom.args = {
   ...Default.args,
   inactiveBackgroundColor: 'var(--red500)',
   activeBackgroundColor: 'var(--purple500)'
```

Comment:
purple ? Please share the screenshot !

Reply:
Akshay this your change in the storybook 😭